src/style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  font-size: 100px;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  flex-direction: column;
+  align-items: flex-start;
+  flex-grow: 1;
+  text-align: center;
+  line-height: 300px;
+  height: 300px;
+  width: 300px;
+}
+
+.block-1 {
+  order: 1;
+  background-color: red;
+}
+
+.block-2 {
+  order: 2;
+  background-color: #c00;
+}
+
+.block-3 {
+  order: 3;
+  background-color: #900;
+}
+
+.block-4 {
+  order: 4;
+  background-color: #600;
+}
+
+.block-5 {
+  order: 5;
+  background-color: #300;
+}
+
+.block-6 {
+  order: 6;
+  background-color: black;
+}
+
+@media (min-width: 600px) {
+  .block-3 {
+    order: 4;
+  }
+
+  .block-4 {
+    order: 3;
+  }
+}
+@media (min-width: 900px) {
+  .container {
+    max-width: 1199px;
```

Comment:
don't restrict container width, it should work properly on big resolutions, try to work with flex-basis